pkg/karmadactl/addons/utils/template_test.go

```diff
@@ -0,0 +1,58 @@
+package utils
+
+import (
+	"testing"
+)
+
+func TestParseTemplate1(t *testing.T) {
+	type args struct {
+		strTmpl string
+		obj     interface{}
+	}
+	tests := []struct {
+		name    string
+		args    args
+		wantErr bool
+		wantNil bool
+	}{
+		{
+			name: "strTmpl and obj are not empty",
+			args: args{
+				strTmpl: "foo",
+				obj:     "bar",
+			},
+			wantErr: false,
+			wantNil: false,
+		},
+		{
+			name: "strTmpl is empty",
+			args: args{
+				strTmpl: "",
+				obj:     "bar",
+			},
+			wantErr: false,
+			wantNil: true,
+		},
+		{
+			name: "obj is nil",
+			args: args{
+				strTmpl: "foo",
+				obj:     nil,
+			},
+			wantErr: false,
+			wantNil: false,
+		},
+	}
+
+	for _, tt := range tests {
+		got, err := ParseTemplate(tt.args.strTmpl, tt.args.obj)
+		if (err != nil) != tt.wantErr {
+			t.Errorf("ParseTemplate() error = %v, wantErr %v", err, tt.wantErr)
+			return
+		}
+
+		if (got == nil) != tt.wantNil {
+			t.Errorf("ParseTemplate() = %v , wantNil %v", got, tt.wantNil)
+		}
```

Comment:
Is it possible to return the content for an equivalent judgment?

Reply:
I will try it later, thx